CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # Changelog
 
+## v1.6.10
+
+Summary of Changes
+------------------
+
+**Minor Changes:**
+* Envoy is updated to release 1.13.2. (Backport PR #12017, Upstream PR #11973, @jrajahalme)
```

Comment:
Sorry, I didn't notice but we have the same thing with Envoy. This should be put with Envoy `1.13.3` messages :-)